kernel/Kconfig

```diff
@@ -479,8 +479,13 @@ config NUM_PIPE_ASYNC_MSGS
 	  Setting this option to 0 disables support for asynchronous
 	  pipe messages.
 
+config MEMPOOL
+	bool "Memory pools"
+	default y
+
 config HEAP_MEM_POOL_SIZE
```

Comment:
add
`if MEMPOOL`
here and enclose all related options.

Reply:
done